.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Thanks a lot for contributing to skops! Please make sure you've taken a look at the contribution guidelines: https://github.com/skops-dev/skops/blob/main/CONTRIBUTING.rst You can find answers to your questions about the checklist below in this file.
+Below is a template you can use to create pull requests. 
```

Comment:
line too long

Reply:
Why do we have to apply line trimming on markdown doc that will appear on github inside PR opening page? I don't get it.